### ISSUE TITLE
[BUG]at the start of a new season database editor just doesn’t work

### ISSUE THREAD

**Issue by Epicforce07** [IUrreta/DatabaseEditor#337]
When I start a new season in f1 manager and try to access database editor with that save, it just disconnects and won’t let me use the application.

**Comment by IUrreta** [comment]
Yeah, don’t send me the save so I can’t take a look at what’s going on 👍

**Comment by Epicforce07** [comment]
Sorry man there’s 2 save files which have the same issue so I thought that it probably wouldn’t be an issue with the saves. Do you still want both of them?

**Comment by x3wolf45** [comment]
as long as you dont have a fix just the previous version that worked fine with me :)

**Comment by Epicforce07** [comment]
[Broken F1M24 Save Files.zip](https://github.com/user-attachments/files/16569625/Broken.F1M24.Save.Files.zip)
There's the save files

**Comment by Epicforce07** [comment]
[log.txt](https://github.com/user-attachments/files/16569629/log.txt)
and log^^^]

**Comment by IUrreta** [comment]
Thanks, will be fixed in the next update
